requirements/test.in

```diff
@@ -5,6 +5,6 @@ pytest-flask
 pytest-sugar
 pytest-cov
 # lets tests run successfully in containers
-fakeredis
+fakeredis==2.16.0
```

Comment:
We never pin dependencies here, not even in the main FlexMeasures requirements. This will easier lead to unresolvable situations.

We should go with `fakeredis < 2.17` and we should also add the root cause. Which is that it requires a redis version we do not have?